source/dem/plane_insertion.cc

```diff
@@ -325,7 +293,7 @@ PlaneInsertion<dim>::insert(
                                            ref_point,
                                            starting_ID_on_proc++,
                                            cell,
-                                           properties_of_one_particle);
+                                           this->particle_properties.at(i++));
```

Comment:
After this, will `this->particle_properties` be used again? If not, would it be useful (for memory) to reset the vector?

Reply:
I think I could remove this attribute from the class and create the vector directly in the insert() function.